src/components/application_manager/test/policy_handler_test.cc

```diff
@@ -3126,6 +3126,22 @@ TEST_F(PolicyHandlerTest, GetEnabledLocalApps_SUCCESS) {
   EXPECT_EQ(enabled_local_apps, policy_handler_.GetEnabledLocalApps());
 }
 
+TEST_F(PolicyHandlerTest, PushAppIdToPTUQueue_PolicyEnabled_SUCCESS) {
+  ChangePolicyManagerToMock();
+  EXPECT_CALL(*mock_policy_manager_, UpdatePTUReadyAppsCount(_));
```

Comment:
@YarikMamykin Why don't we check apps count?

Reply:
@AByzhynar added check in https://github.com/smartdevicelink/sdl_core/pull/3292/commits/73964199f3149237d32109dbd2f17a3e8e56a8dd

Reply:
@YarikMamykin Is it possible to check parameter of `UpdatePTUReadyAppsCount(_)` insted of `_` using?

Reply:
@AByzhynar done in https://github.com/smartdevicelink/sdl_core/pull/3292/commits/55f94fb5c04ed4edfe064d13dfd0de6d18e53395